src/utils/useStylesInterface.js

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react';
+
+import WithStylesContext from '../WithStylesContext';
+import { _getInterface } from '../ThemedStyleSheet';
+
+export default function useStylesInterface() {
+  const { stylesInterface } = useContext(WithStylesContext);
```

Comment:
should this use https://github.com/airbnb/react-with-styles/pull/224/files#diff-56c9bb27cfc316ce0cfd3db74a78d2e0R18-R20?